parse_date.re

```diff
@@ -2159,6 +2159,8 @@ timelib_time *timelib_parse_from_format_with_map(const char *format, const char
 	int          iso_year = TIMELIB_UNSET;
 	int          iso_week_of_year = TIMELIB_UNSET;
 	int          iso_day_of_week = TIMELIB_UNSET;
+	int          week_of_year = TIMELIB_UNSET;
+	int          day_of_week = TIMELIB_UNSET;
```

Comment:
These are defined, and used while parsing. But nothing is done with these two variables after they are assigned.